### PR TITLE
test: Add more tests for workflow/credential migrations when enabling UM

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

File: packages/design-system/src/components/N8nMenuItem/MenuItem.vue [hunk 1]
```diff
@@ -30,6 +30,7 @@
 					[$style.disableActiveStyle]: !isItemActive(child),
 					[$style.active]: isItemActive(child),
 				}"
+				:data-test-id="`menuitem-${child.id}`"
```
Comment (OlegIvaniv) [inline packages/design-system/src/components/N8nMenuItem/MenuItem.vue:33]: I'm not a huge fan of adding unique ids to the test-id identifier. While it's easier to target an element this way, it prevents us from getting all `menuitem` if we'd need to count them or iterate over them in any other way. To get the individual menu item in cypress could be done like `cy.getByTestId('menuitem-credentials').contains($id).parents('[data-test-id="workflow-card"]')`

File: cypress/e2e/3-default-owner.cy.ts [hunk 2]
```diff
@@ -1,9 +1,115 @@
...
-	// todo test for adding workflow
-	// todo test for setting up UM again through settings
-	// todo test that workflows migrated successfully
+	it('should be able to use n8n without user management and setup UM', () => {
+		describe('should skip owner setup', () => {
+			cy.skipSetup();
+			cy.url().should('include', workflowsPage.url);
+		});
+
+		describe('should be able to create workflows', () => {
```
Comment (OlegIvaniv) [inline cypress/e2e/3-default-owner.cy.ts:39]: Do we need this? Wouldn't this already be tested by the `workflows` spec? Or perhaps we can move the missing bits to this spec to also test reload: 
```
cy.reload();

workflowPage.getters.workflowNameInput().should('contain.value', 'Test workflow');
```

**Review by OlegIvaniv — COMMENTED** [review]
Nice one 👌 Left just two minor comments

File: cypress/e2e/3-default-owner.cy.ts (hunk 2 above)
Comment (mutdmour) [reply cypress/e2e/3-default-owner.cy.ts:39]: this is in the special case of no user management... we have run into bugs before we forget users don't have this setup... that's why I am covering it here again. and the reload.. don't want the user to be redirected to sign in or anything

File: packages/design-system/src/components/N8nMenuItem/MenuItem.vue (hunk 1 above)
Comment (mutdmour) [reply packages/design-system/src/components/N8nMenuItem/MenuItem.vue:33]: but the id here is `credentials` so the test id is `menuitem-credentials`

**Comment by OlegIvaniv** [comment]
Also, CI is failing on outdated Actionbox snapshot, they should be regenerated

**Review by OlegIvaniv — DISMISSED** [review]

**Pull request merged**